hathor/cli/grafana_dashboard.py

```diff
@@ -32,28 +34,29 @@
 
 # All charts json
 CHARTS_ARRAY = [
-    'transactions',
-    'blocks',
-    'orphan_blocks',
-    'avg_seconds_per_block',
-    'tx_rate',
-    'hash_rate_stacked',
-    'blocks_weight',
-    'send_token_timeouts',
-    'peers',
-    'cpu',
-    'cpu_load1m',
-    'cpu_load5m',
-    'ram_percent',
-    'ram_line',
-    'network',
-    'cpu_line',
-    'load', 'disk',
-    'websocket_connections',
-    'subscribed_addresses',
-    'stratum_completed_jobs',
-    'stratum_blocks_found',
-    'stratum_estimated_hash_rate'
+    (True, 'transactions'),
+    (True, 'blocks'),
+    (True, 'orphan_blocks'),
+    (True, 'avg_seconds_per_block'),
+    (True, 'tx_rate'),
+    (True, 'hash_rate_stacked'),
+    (True, 'blocks_weight'),
+    (True, 'send_token_timeouts'),
+    (True, 'peers'),
+    (False, 'cpu'),
+    (False, 'cpu_load1m'),
+    (False, 'cpu_load5m'),
+    (False, 'ram_percent'),
+    (False, 'ram_line'),
+    (False, 'network'),
+    (False, 'cpu_line'),
+    (False, 'load'),
+    (False, 'disk'),
+    (True, 'websocket_connections'),
+    (True, 'subscribed_addresses'),
+    (True, 'stratum_completed_jobs'),
+    (True, 'stratum_blocks_found'),
+    (True, 'stratum_estimated_hash_rate'),
```

Comment:
Continue discussion from https://gitlab.com/HathorNetwork/hathor-python/-/merge_requests/412#note_357974801

I like @pedroferreira1's suggestion. I'll make an `enum.Flag` for this.